docs/modules/con-proxy-overview.adoc

```diff
@@ -249,11 +246,8 @@ _Transparent_ multi-tenancy means each tenant has the illusion of having their o
 // TODO we probably don't need the level of detail below, just summarize
 // and provide the detail in the deploying section
 
-We can further classify deployment topologies in how many proxy instances are used. For example:
-
-Single proxy instance::
-
-Proxy pool::
-//-
+We can further classify deployment topologies in how many proxy instances are used. 
+For example:
 
-== More about filters
+* Single proxy instance
```

Comment:
Would it be clearer to describe `Single Proxy instance` as `Sidecar` ? 

We also miss embedded from the list (though I think thats intentional)